ios/Classes/VoximplantPlugin.m

```diff
@@ -100,5 +106,67 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     }
 }
 
+#pragma mark - FlutterStreamHandler
+
+- (FlutterError * _Nullable)onListenWithArguments:(id _Nullable)arguments eventSink:(nonnull FlutterEventSink)events {
+    if ([arguments isKindOfClass:[NSString class]]) {
+        NSString *type = (NSString *)arguments;
+        if ([type isEqual:@"logs"]) {
+            self.logsEventSink = events;
+        }
+    }
+    return nil;
+}
+
+- (FlutterError * _Nullable)onCancelWithArguments:(id _Nullable)arguments {
+    if ([arguments isKindOfClass:[NSString class]]) {
+        NSString *type = (NSString *)arguments;
+        if ([type isEqual:@"logs"]) {
+            self.logsEventSink = nil;
+        }
+    }
+    return nil;
+}
+
+#pragma mark - VILogDelegate
+
+- (void)didReceiveLogMessage:(NSString *)message severity:(VILogSeverity)severity {
+    if (self.logsEventSink) {
+        
+        self.logsEventSink(@{
```

Comment:
Flutter requires to call all method channel API on the main queue, but Voximplant iOS SDK provides the log messages on a separate queue.

should not be a dispatch to the main queue here?